frontend/src/metabase/query_builder/components/notebook/NotebookNativePreview/NotebookNativePreview.tsx

```diff
@@ -39,7 +39,7 @@ export const NotebookNativePreview = (): JSX.Element => {
   const { data, error, isFetching } = useGetNativeDatasetQuery(payload);
 
   const showLoader = isFetching;
-  const showError = !isFetching && canRun && error;
+  const showError = !isFetching && canRun && (error as object);
```

Comment:
if `error` is of type `object` then it would always be truthy. i'm guessing this is a bad type cast. might be better to use `unknown`?

Reply:
I must have been half asleep here. We just need error to convert error to a boolean haha